charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 2.4.4
+version: 2.5.0
```

Comment:
one final question:

Based on the information from the [When to increase a chart’s version number? section](https://www.eclipse.org/packages/contribute/) of the IoT Packages contribution guide, I would think that increasing the patch version should be sufficient for this kind of change, or am I missing something?

@calohmn WDYT?

Reply:
The changes in this PR change the name of the Hono resources if the chosen _release name_ doesn't contain "hono" (or the corresponding `nameOverride` for hono).
So, using the "eclipse-hono" release name (as documented in the Hono chart README), there would be no change to most of the names (only the Kafka related resources, I think).

But, when deploying Hono as part of the cloud2edge package, using `c2e` as release name, the names will change from
```
c2e-adapter-amqp
c2e-service-auth
c2e-ditto-gateway
c2e-ditto-nginx
```
to
```
c2e-hono-adapter-amqp
c2e-hono-service-auth
c2e-ditto-gateway
c2e-ditto-nginx
```
So, this is an incompatible change, requiring adaptations to the cloud2edge chart. That's why we have restricted the Hono chart version in the cloud2edge dependencies to `~2.4.0`.

So, we have to (at least) increase the Hono chart minor version here. (See also discussion in https://github.com/eclipse/packages/pull/416#discussion_r1001436531.)